atmos_model.F90

```diff
@@ -262,7 +262,7 @@ subroutine update_atmos_radiation_physics (Atmos)
       if (ierr/=0)  call mpp_error(FATAL, 'Call to CCPP timestep_init step failed')
 
       if (GFS_Control%do_sppt .or. GFS_Control%do_shum .or. GFS_Control%do_skeb .or. &
-          GFS_Control%lndp_type > 0  .or. GFS_Control%do_ca ) then
+          GFS_Control%lndp_type > 0  .or. GFS_Control%do_ca .or. GFS_Control%do_spp) then
```

Comment:
Probably a stupid question, but ... what is the difference between spp and sppt?

Reply:
SPP is Stochastically Perturbed Parameterizations, which perturbs specific parameters within physics parameterizations; in this instance, in a number of RAP/HRRR-based parameterizations.  SPPT is Stochastic Perturbed Parameterization Tendencies, which perturb all summed tendencies right after the physics calls.  The latter is known as an 'ad-hoc' scheme, since it's more-or-less a white noise technique added after the fact, while the former is much more targeted, and "active", since the perturbations occur within the physics parameterizations themselves.

Reply:
Thanks for educating me. What a difference one letter can make.